Lib/test/test_typing.py

```diff
@@ -9464,5 +9464,11 @@ def test_is_not_instance_of_iterable(self):
             self.assertNotIsInstance(type_to_test, collections.abc.Iterable)
 
 
+def load_tests(loader, tests, ignore):
```

Comment:
This parameter is called `pattern` in every other `load_tests` function in `Lib/test/`

```suggestion
def load_tests(loader, tests, pattern):
```